Makefile.PL

```diff
@@ -102,17 +102,30 @@ my %opts = (
 $opts{CAPI} = 'TRUE' if $Config{archname} =~ /-object\b/i;
 
 if (my $gccversion = $Config{gccversion}) {	# ask gcc to be more pedantic
-    warn "WARNING: Your GNU C $gccversion compiler is very old. Please upgrade it and rebuild perl.\n"
-	if $gccversion =~ m/^\D*(1|2\.[1-8])/;
-    print "Your perl was compiled with gcc (version $Config{gccversion}), okay.\n";
-    $gccversion =~ s/[^\d\.]//g; # just a number please
-    $opts{DEFINE} .= ' -W -Wall -Wpointer-arith -Wbad-function-cast';
-    $opts{DEFINE} .= ' -Wno-comment -Wno-sign-compare -Wno-cast-qual';
-    $opts{DEFINE} .= ' -Wmissing-noreturn -Wno-unused-parameter' if $gccversion ge "3.0";
-    if ($is_developer && $::opt_g) {
-        $opts{DEFINE} .= ' -DPERL_GCC_PEDANTIC -ansi -pedantic' if $gccversion ge "3.0";
-        $opts{DEFINE} .= ' -Wdisabled-optimization -Wformat'    if $gccversion ge "3.0";
-        $opts{DEFINE} .= ' -Wmissing-prototypes';
+    if ($gccversion =~ m/ Clang \d/) {
```

Comment:
I think that should be `m/ clang ([0-9][-0-9.]*)/i` to be more future proof

Reply:
Indeed. Updated in 624c6ce1e4b259223040014149b7d7aea4a48ab0